uber/models.py

```diff
@@ -1144,16 +1144,14 @@ def badge_cost(self):
 
     @property
     def age_group_conf(self):
-        if self.age_group and self.age_group != c.AGE_UNKNOWN:
-            return c.AGE_GROUP_CONFIGS[self.age_group]
-        elif self.birthdate:
+        if self.birthdate:
             day = c.EPOCH.date() if date.today() <= c.EPOCH.date() else sa.localized_now().date()
             attendee_age = (day - self.birthdate).days / 365.2425
```

Comment:
this /365 here should be converted to some kind of native date handling thing to deal with leap years and junk more precisely.  discussion in https://github.com/magfest/ubersystem/issues/1125, I'll add a note there.

don't hold up this PR for it though, it's just something I noticed